From: Jan Luebbe <jluebbe@debian.org>
Date: Wed, 29 Aug 2012 20:06:27 +0200
Subject: [PATCH] Release raspberrypi/20120829

Signed-off-by: Jan Luebbe <jluebbe@debian.org>
---
 Makefile |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index bdf851f..1b03a99 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 VERSION = 3
 PATCHLEVEL = 2
 SUBLEVEL = 27
-EXTRAVERSION =
+EXTRAVERSION =-20120829
 NAME = Saber-toothed Squirrel
 
 # *DOCUMENTATION*
