From: popcornmix <popcornmix@gmail.com>
Date: Thu, 7 Feb 2013 16:17:03 +0000
Subject: [PATCH] Default force_bulk to false

---
 sound/arm/bcm2835-vchiq.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/arm/bcm2835-vchiq.c b/sound/arm/bcm2835-vchiq.c
index 33b7481..9ecb2d6 100755
--- a/sound/arm/bcm2835-vchiq.c
+++ b/sound/arm/bcm2835-vchiq.c
@@ -60,7 +60,7 @@ typedef struct opaque_AUDIO_INSTANCE_T {
 	short peer_version;
 } AUDIO_INSTANCE_T;
 
-bool force_bulk = 1;
+bool force_bulk = false;
 
 /* ---- Private Variables ---------------------------------------------------- */
 
