From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Sat, 6 Apr 2013 15:00:46 +0200
Subject: [PATCH] Release raspberrypi/20130406

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 50dc4d1..e90f629 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 VERSION = 3
 PATCHLEVEL = 6
 SUBLEVEL = 11
-EXTRAVERSION =
+EXTRAVERSION =-20130406
 NAME = Terrified Chipmunk
 
 # *DOCUMENTATION*
